chapters/connectors.tex

```diff
@@ -904,7 +904,16 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
 \subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
 
 Before connection equations are generated, the virtual connection graph is transformed into a set of spanning trees by removing optional spanning tree edges from the graph.
-This is performed in the following way:
+
+Before that processing any redundant breakable branches are removed in each connection set:
```

Comment:
Can the before-the-before be avoided by putting this paragraph above the previous one?

For example, starting with something like:
```suggestion
As a preparation of generating connection graph equations, any redundant breakable branches are removed in each connection set:
```